openroad_interface/parasitic_estimation.py

```diff
@@ -0,0 +1,44 @@
+from var import directory
+import re
+import math
+
+
+def total_euclidean_distance(net_list, coord_data, unit):
+    the_component = net_list[0]
+    result = 0.0
+    for x in range(1, len(net_list)):
+        result += math.sqrt(pow(coord_data[the_component]["x"]/unit - coord_data[net_list[x]]["x"]/unit, 2) + pow(coord_data[the_component]["y"]/unit - coord_data[net_list[x]]["y"]/unit, 2))
+    return result
+
+def parasitic_estimation(lef_pitch, layer_res, layer_cap, units):
```

Comment:
Add function docs. What does the function do, what are the inputs, what data type. 
Better yet, add the types in the signature, python3 style, along with a return type

https://docs.python.org/3/library/typing.html

Reply:
```
def parasitic_estimation(lef_pitch: float, layer_res: float, layer_cap: float, units: float) -> Dict():
    '''
        Calculate approximate R, C, wirelength for nets.

params: 
    lef_pitch: ....
    layer_res: .....

    '''